docs-source/content/developerguide/requirements.md

```diff
@@ -8,8 +8,8 @@ weight: 1
 In addition to the requirements listed in the [User guide]({{< relref "/userguide/introduction/introduction.md#prerequisites" >}}), the following software is also required to obtain and build the operator:
 
 * Git (1.8 or later recommended)
-* Java Developer Kit (1.8u131 or later recommended; please use 1.8, tests will not work on 1.9 or later versions)
-* Apache Maven (3.3 or later recommended)
+* Java Developer Kit (11 required, 11.0.2 recommended)
+* Apache Maven (3.5.3 min, 3.6 recommended)
 
 The operator is written primarily in Java, BASH shell scripts, and WLST scripts.  The Java code uses features introduced in Java 1.8 -- for example, closures -- but does not use any Java 1.9 features.
```

Comment:
Can you please remove the second sentence?  This is true today, but need not be true going forward now that we require JDK 11.